test/performance/odp_l2fwd.c

```diff
@@ -789,6 +789,10 @@ static int create_pktio(const char *dev, int idx, int num_rx, int num_tx,
 		config.pktout.bit.tcp_chksum_ena  = 1;
 	}
 
+	/* Provide hint to pktio that packet references are not used */
+	if (pktio_capa.config.pktout.bit.no_packet_refs)
```

Comment:
The capability check here is required by the current API, but I feel it is a bit silly that it has to be done. This setting is just making life easier to an ODP implementation and every implementation obviously has to support sending non-referenced packets anyway.

Maybe we could add a note in odp_pktout_config_opt_t that it is always ok to set the no_packet_refs flag, i.e. every ODP implementation supports it?

Reply:
Ok. I'll add a comment on top of odp_pktout_config_opt_t:no_packet_refs field mentioning the same.